docker-registry-sync/reposync/src/reposync/dregsy_config.py

```diff
@@ -90,8 +133,26 @@ def create_dregsy_yamls(stages: List[Stage]) -> List[Tuple[str, str]]:
             target_auth = encode_credentials(
                 to_obj.destination.user, to_obj.destination.password
             )
+            requires_option = (
+                "" if stage.depends_on is None else f"(requires {stage.depends_on})"
```

Comment:
```suggestion
                "" if not stage.depends_on else f"(requires {stage.depends_on})"
```

Reply:
I specifically coded it like this to support empty list "[]" and print "(requires [])".